user/views.py

```diff
@@ -1,18 +1,17 @@
-from datetime import datetime, timedelta
-from json     import JSONDecodeError
-from jwt      import DecodeError
+from datetime           import datetime, timedelta
+from json               import JSONDecodeError
+from jwt                import DecodeError
 
-import bcrypt
-import json
-import jwt
-import re
+import bcrypt, json, jwt, re
```

Comment:
```suggestion
import bcrypt
import json
import jwt
import re
from datetime import datetime, timedelta
from json     import JSONDecodeError              
from jwt      import DecodeError
```
저는 이게 조금 더 깔끔해 보입니다.(의견)